libpod/runtime.go

```diff
@@ -225,11 +225,16 @@ func (r *Runtime) Shutdown(force bool) error {
 		}
 	}
 
-	_, err := r.store.Shutdown(force)
-	if err != nil {
-		return err
+	var lastError error
+	if _, err := r.store.Shutdown(force); err != nil {
+		lastError = errors.Wrapf(err, "Error shutting down container storage")
+	}
+	if err := r.state.Close(); err != nil {
+		if lastError != nil {
+			logrus.Errorf("%v", lastError)
+		}
+		lastError = err
```

Comment:
This maybe golang education time for me, but a lot of these calls to this function are:
```
defer runtime.Shutdown(false)
```
Will this second lastError get printed in that instance or do we need a second logrus.Errorf statement here?

Reply:
I think this is more an error in the defer statement than it is one in the library - if the error is being thrown out there's not much we can do.
I thought Golint would catch ignored returns in deferred functions, but apparently not.
Should see if there's a way to enable that.